docs/kr/README.md

```diff
@@ -198,11 +198,26 @@ docker exec palworld-server backup
 
 rcon이 활성화된 경우 서버는 백업 전에 저장을 실행합니다.
 
+## 백업 복원 하기
+
+백업을 복원하려면 다음 명령을 사용합니다:
+
+```bash
+docker exec -i palworld-server restore
```

Comment:
```suggestion
docker exec -it palworld-server restore
```

I really thought only `-i` was needed but that resulted in this
```
Backup List:
1) /palworld/backups/blah.tar.gz
2) /palworld/backups/palworld-save-2024-02-02_00-21-18.tar.gz
3) /palworld/backups/subfolder/palworld-save-2024-02-02_00-20-50.tar.gz
#? 3
Selected backup: /palworld/backups/subfolder/palworld-save-2024-02-02_00-20-50.tar.gz
Do you want to continue with the command?
y
```

Changing it to `-it` gave the expected

```
Backup List:
1) /palworld/backups/blah.tar.gz
2) /palworld/backups/palworld-save-2024-02-02_00-21-18.tar.gz
3) /palworld/backups/subfolder/palworld-save-2024-02-02_00-20-50.tar.gz
#? 3
Selected backup: /palworld/backups/subfolder/palworld-save-2024-02-02_00-20-50.tar.gz
Do you want to continue with the command?
When you run it, the server will be stopped and the recovery will proceed. (y/n): y
```